kolibri/core/assets/src/vue/content-renderer/download-button.vue

```diff
@@ -6,15 +6,15 @@
       class="dropdown-button"
       @click="toggleDropdown"
       aria-haspopup="true">
-      <span class="dropdown-button-text" :class="{uparrow: dropdownopen}">
+      <span class="dropdown-button-text" :class="{'dropdown-button-text-open': dropdownOpen}">
         {{ $tr('downloadContent') }}
       </span>
     </button>
     <ul
       ref="dropdownitems"
       class="dropdown-items"
-      :class="{ dropdownopen: dropdownopen }"
-      :aria-hidden="dropDownOpenText"
+      :class="{ 'dropdown-open': dropdownOpen }"
+      :aria-hidden="dropdownOpenText"
```

Comment:
There is a lot of horribleness that would go away if we drop IE10 as well, so I am all for it. @jamalex is that what we decided?